### PR TITLE
Closes #14 | Create dataset loader for ara-close-lange

### LINKED ISSUE #14: Create dataset loader for ara-close-lange
Dataloader name: `ara_close/ara_close.py`
DataCatalogue: http://seacrowd.github.io/seacrowd-catalogue/card.html?ara_close

|     Dataset| ara_close |
|-------------|---|
| Description | The dataset contribution of this study is a compilation of short fictional stories written in Bikol for readability assessment. The data was combined other collected Philippine language corpora, such as Tagalog and Cebuano.  The data from these languages are all distributed across the Philippine elementary system's first three grade levels (L1, L2, L3). We sourced this dataset from Let's Read Asia (LRA), Bloom Library, Department of Education, and Adarna House. |
| Subsets | - |
| Languages | bcl, tgl, ceb |
| Tasks | Readibility Assessment |
| License | Creative Commons Attribution 4.0 (cc-by-4.0) |
| Homepage | https://github.com/imperialite/ara-close-lang |
| HF URL | - |
| Paper URL | https://aclanthology.org/2023.findings-acl.331/ |

### PR DESCRIPTION
**Pull request by MJonibek**
Closes #14 

Note:

1. In the data card, there are three languages: bcl, tgl, ceb. However, data source for [tgl was deleted](https://github.com/SEACrowd/seacrowd-datahub/issues/14#issuecomment-1869864418).
2. READABILITY_ASSESSMENT [was added to Tasks constants](https://github.com/SEACrowd/seacrowd-datahub/issues/14#issuecomment-1869240572).

Please name your PR after the issue it closes. You can use the following line: "Closes #ISSUE-NUMBER" where you replace the ISSUE-NUMBER with the one corresponding to your dataset.

### Checkbox
- [x] Confirm that this PR is linked to the dataset issue.
- [x] Create the dataloader script `seacrowd/sea_datasets/my_dataset/my_dataset.py` (please use only lowercase and underscore for dataset naming).
- [x] Provide values for the `_CITATION`, `_DATASETNAME`, `_DESCRIPTION`, `_HOMEPAGE`, `_LICENSE`, `_URLs`, `_SUPPORTED_TASKS`, `_SOURCE_VERSION`, and `_SEACROWD_VERSION` variables.
- [x] Implement `_info()`, `_split_generators()` and `_generate_examples()` in dataloader script.
- [x] Make sure that the `BUILDER_CONFIGS` class attribute is a list with at least one `SEACrowdConfig` for the source schema and one for a seacrowd schema.
- [x] Confirm dataloader script works with `datasets.load_dataset` function.
- [x] Confirm that your dataloader script passes the test suite run with `python -m tests.test_seacrowd seacrowd/sea_datasets/<my_dataset>/<my_dataset>.py`.
- [ ] If my dataset is local, I have provided an output of the unit-tests in the PR (please copy paste). This is OPTIONAL for public datasets, as we can test these without access to the data files.

### CODE REVIEW

**Comment by sabilmakbar** [comment]
Hi @MJonibek, thanks for working on this dataloader. One req: can you separate the config changes in different PR; leaving this as Dataloader PR only? Thanks!

File: seacrowd/utils/constants.py
Comment (ljvmiranda921) [inline seacrowd/utils/constants.py:1]: Echoing @sabilmakbar 's comment, let's separate these changes in a different PR.
Also, might be better to limit the changes to the new task you're adding (don't include the format and isort changes).

File: seacrowd/sea_datasets/ara_close/ara_close.py
Comment (ljvmiranda921) [inline seacrowd/sea_datasets/ara_close/ara_close.py:118]: Loading all languages in a single config is now optional. I think it's safe to remove this and just focus on the two language subsets.

File: seacrowd/sea_datasets/ara_close/ara_close.py
Comment (ljvmiranda921) [inline seacrowd/sea_datasets/ara_close/ara_close.py:216]: Might need second opinion from another reviewer (cc: @gentaiscool ), should the `id` here be restricted to ordinal numbers? (e.g., 0, 1,2, 3, 4). Is the `title` acceptable here? The test errors out when this runs (IDs aren't globally unique) so I don't think we should use the `title` imo

**Review by ljvmiranda921 — CHANGES_REQUESTED** [review]
Hi @MJonibek thanks for the PR! Some high-level comments:

- A config that loads all languages is now optional. I recommend just removing that.
- The `ara_close_ceb_*` config fails because the IDs aren't globally unique. I don't think we should use `title` for the`id`.
- There's a missing `__init__.py` file
- Let's put the changes in `constants.py` in another PR. Remove that commit here and transfer it into another branch then make a new PR

File: seacrowd/sea_datasets/ara_close/ara_close.py
Comment (ljvmiranda921) [inline seacrowd/sea_datasets/ara_close/ara_close.py:97]:
```suggestion
class AraCloseDataset(datasets.GeneratorBasedBuilder):
```

**Comment by sabilmakbar** [comment]
Hi @MJonibek. Thanks for implementing the dataloader and proposing to add `READABILITY ASSESSMENT` task (and implementing it on ur own). Let's split it into another PR for the cleanliness of Config vs. Dataloader PRs. Thanks!

**Comment by MJonibek** [comment]
Thank you for your comment, I will work on it today

**Comment by MJonibek** [comment]
I transformed changes in constants.py into separate PR ([292](https://github.com/SEACrowd/seacrowd-datahub/pull/292))

Changes in ara_close.py:

1. Deleted config that loads all languages
2. Change id from title to integers

**Review by ljvmiranda921 — APPROVED** [review]
LGTM!

File: seacrowd/sea_datasets/ara_close/ara_close.py
Comment (gentaiscool) [reply seacrowd/sea_datasets/ara_close/ara_close.py]: I agreed it has to be ordinal number

**Review by gentaiscool — APPROVED** [review]
I have checked your changes. LGTM

**Pull request merged**
